a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -363,3 +363,70 @@ def revert(self, vthunder, *args, **kwargs):
         except Exception as e:
             LOG.exception("Failed to revert partition create : %s", str(e))
             raise
+
+
+class CreateVLAN(task.Task):
+
+    """Task to create VLAN on vThunder device"""
+
+    default_provides = a10constants.VE_INTERFACE
+
+    @axapi_client_decorator
+    def execute(self, vlan_id, tag_interface, vthunder):
```

Comment:
What if I want to use a NIC in a higher slot number w/ faster throughput?

Reply:
Had thought of having a map in the vThunder config to map Interface to VLAN IDs. Thought could take it up as a story later. If you wan't it addressed with this I can do it.

Reply:
Yup that's I had in mind too. Let's address the change in this PR. We can have it default to using eth1. But the user can provide a NIC -> VLAN mapping in the config.

Reply:
Ok. Will take care of this.

Reply:
Done.